discovery-provider/integration_tests/queries/test_get_attestation.py

```diff
@@ -126,10 +126,7 @@ def test_get_attestation(app):
 def patch_get_all_other_nodes():
     with patch(
         "src.queries.get_attestation.get_all_other_discovery_nodes_cached",
-        return_value=(
-            ["some_discovery.com"],
-            ["0x94e140D27F3d5EE9EcA0109A71CcBa0109964DCa"],
```

Comment:
Is this supposed to be changing?

Reply:
yea, b/c the function doesn't return this mocked return value, it returns just a list of endpoints

Reply:
actually, this method needs the wallets, updated accordingly